actors/builtin/miner/miner_actor.go

```diff
@@ -1475,9 +1491,13 @@ func (a Actor) AddLockedFund(rt Runtime, amountToLock *abi.TokenAmount) *adt.Emp
 			rt.Abortf(exitcode.ErrInsufficientFunds, "insufficient funds to lock, available: %v, requested: %v", unlockedBalance, *amountToLock)
```

Comment:
@anorth is this still an issue?

Reply:
Yes, this is recently added. Maybe the comment is hard to grok, but the requirement that you can't lock more than you have must be checked.